Dockerfile

```diff
@@ -69,7 +69,7 @@ COPY . .
 
 RUN mkdir build_in_ci && cd build_in_ci && \
     cmake                                          \
-      -D CMAKE_BUILD_TYPE=Release                  \
+      -D CMAKE_BUILD_TYPE=Debug                    \
```

Comment:
You want to build the debug version of  DALI?

Reply:
This `Dockerfile` is made for development reasons (as it is stated in the comment on top of it): it installs packages like `gdb` and installs DALI's nightly version, so it'd be rational to have debug version of `dali_backend` also. I would envision having debug version of DALI itself here too, but that's more complicated at this point, so it's only dali_backend for now.